nucypher/blockchain/eth/actors.py

```diff
@@ -625,13 +626,16 @@ def __init__(self,
             self.stakes = StakeList(registry=self.registry, checksum_address=self.checksum_address)
```

Comment:
Btw just an idea... What if we get the `checksum_address` from blockchain when we're configuring the worker? (currently it's from geth)

Reply:
Yeah, I think this is a good Idea.